dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG PYTHON_VERSION=3.9
+
+FROM python:${PYTHON_VERSION} as builder
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
```

Comment:
In this stage no need to set the env value for `PYTHONUNBUFFERED=1`. It doesn't use at this stage